pkg/cmd/server/logs.go

```diff
@@ -33,22 +41,88 @@ var logsCmd = &cobra.Command{
 			query = "follow=true"
 		}
 
-		ws, res, err := apiclient.GetWebsocketConn("/log/server", &activeProfile, &query)
-		if err != nil {
-			log.Fatal(apiclient.HandleErrorResponse(res, err))
-		}
+		switch {
+		case fileFlag:
+			readServerLogFile()
+
+		default:
+
```

Comment:
Just notice the extra newline 😅 , please remove it and we're good to go.

Reply:
no problem

Reply:
the one before or after default

Reply:
on line 49 (after default)